test/utils/session/SessionFactory.ts

```diff
@@ -1,36 +1,42 @@
 import { Session } from 'ch-node-session-handler/lib/session/model/Session';
 import { generateSessionId, generateSignature } from 'ch-node-session-handler/lib/utils/CookieUtils';
 
```

Comment:
Unfortunately, due to formatting the whole file shows up as changed, I'll highlight the changes:
1. Constructor now has two more parameters for specifying user permissions and admin.
2. Update the permissions value and the admin_permissions according to those new parameters.